Mathlib/Data/Ordmap/Ordnode.lean

```diff
@@ -666,17 +709,19 @@ protected def copair {β} (t₁ : Ordnode α) (t₂ : Ordnode β) : Ordnode (Sum
 
 /- warning: ordnode.pmap -> Ordnode.pmap is a dubious translation:
```

Comment:
You can remove dubious translations warnings, but I believe it's good to add the `ₓ` in the `#align` as suggested